zstacklib/zstacklib/utils/gpu.py

```
@@ -996,3 +1015,549 @@ def parse_alibaba_ppu_metric_output(output):
+                result.update(gpu_info.extra)
+                results.append(result)
+
+        if results:
+            return results
+    except Exception as e:
+        logger.debug("Plugin failed, fallback to legacy: %s" % str(e))
+
+    return results
+
+
+def get_metrics(pci_address=None, pci_device=None, vendor_name=None):
```

Ruff (0.14.13)

Unused function argument: `pci_address`

(ARG001)